app/helpers/compare_locales_helper.rb

```diff
@@ -0,0 +1,43 @@
+module CompareLocalesHelper
+  include ApplicationHelper
+
+  LOCALE_1 = "./config/locales/en.yml"
+  LOCALE_2 = "./config/locales/es.yml"
```

Comment:
We will also have to run these tests against all files including `ptbr.yml` and any future translations. We should use regex to access these files (something like `"./config/locales/*.yml"`)